notebooks/source/bayesian_hierarchical_stacking.ipynb

```diff
@@ -800,7 +800,7 @@
     "    N = X.shape[0]\n",
     "    d = X.shape[1]\n",
     "    N_test = X_test.shape[0]\n",
-    "    K = lpd_point.shape[1]  # number of candidate models\n",
+    "    K = exp_lpd_point.shape[1]  # number of candidate models\n",
```

Comment:
I think we need to use `lpd_point` as the input of the model, rather than `exp_lpd_point`. Later, we will take exponential of `lpd_point` at `jax.nn.logsumexp(lpd_point + log_w, axis=1)`

Reply:
Ok, I see. I think it is enough to change the name at the function definition.

Reply:
You are right. It seems that we don't use `exp_lpd_point` in the colab. Do you want to fix the typo in this PR?

Reply:
I am trying to fix it, but, it is already fixed. I do not know if it was my pull request or the request of another contributor.  I send a message through the issue.